streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractWindowBytesStoreTest.java

```diff
@@ -710,73 +710,9 @@ public void testPutAndFetchAfter() {
         assertNull(entriesByKey.get(6));
     }
 
-    @Test
-    @SuppressWarnings("deprecation")
-    public void testPutSameKeyTimestamp() {
```

Comment:
Why removing the whole test?

Reply:
My mistake, I thought it was repeated. Will add it back.